### DISCUSSION TITLE
Looking for FastCRUD Maintainers🚀

### DISCUSSION THREAD

**Discussion by igorbenav** [igorbenav/fastcrud#154] [Announcements]
🚀 **Exciting News**! With FastCRUD surpassing **600 stars** and **38k downloads** 🎉, we’re thrilled by the community's response! However, the influx of issues and PRs is growing faster than I can manage alone.

This post officially kicks off the search for new maintainers to join the FastCRUD project. I’ve already extended an invite to @JakNowy, who has been actively contributing and practically taking on this role. Additionally, I’d love to hear if @dubusster, @feluelle, or @slaarti are interested in stepping up.

If you’re passionate about FastCRUD and weren’t mentioned, don’t worry! Please express your interest in this thread and start contributing through PRs or other helpful actions.

Looking forward to working with you to make FastCRUD an even better way to prototype FastAPI!

**Comment by natehouk** [comment]
Interested, count me in

**Comment by feluelle** [comment]
I would be interested in general, but I am afraid I have the time to be a consistent maintainer.

**Comment by JakNowy** [comment]
Hello, sorry I have been offline for some days, I think the invitation already exipred for me. I am willing to keep helping, but not sure if I can provide responsible maintainance, probably depends on the support of rest of community.

**Comment by suyeon-jung-dev** [comment]
Hello, I am currently using FastAPI at my company to develop a prototype server for an LLM-based API. While using methods like `update` and others, I noticed areas that could be improved. I would like to contribute to making these improvements if possible.
